.circleci/scripts/binary_populate_env.sh

```diff
@@ -102,7 +102,7 @@ PIP_UPLOAD_FOLDER='nightly/'
 # We put this here so that OVERRIDE_PACKAGE_VERSION below can read from it
 export DATE="$(date -u +%Y%m%d)"
 #TODO: We should be pulling semver version from the base version.txt
-BASE_BUILD_VERSION="1.12.0.dev$DATE"
+BASE_BUILD_VERSION="1.13.0.dev$DATE"
```

Comment:
Is this file even used for anything?
```suggestion
BASE_BUILD_VERSION="1.14.0.dev$DATE"
```